mecanum_drive_controller/test/diff_drive_test.cpp

```diff
@@ -0,0 +1,151 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
There is certainly a lot of work to migrate all these tests for the mecanum drive.
I'll suggest to provide some basic tests starting from this one.

Anyway, all the tests are rostests, which depend on gazebo. So, given the issues with simulating mecanum or omni wheels on gazebo, that looks like a big blocking issue, right?